### PR TITLE
Adding scenario of jobseeker for 18 and 19 year olds

### LINKED ISSUE #56: Bug: Jobseeker support age test linked to child
The legislation has age for this linked to whether there is a dependent child:

age must be 18 is no dependent child
otherwise, must be 20

http://legislation.govt.nz/act/public/1964/0136/latest/DLM5478526.html?search=ts_act%40bill%40regulation%40deemedreg_social+security_resel_25_a&p=1

Is this because 18 and 19 year old are eligible for Young Parent Payment? Please ensure YPP is included.

### PR DESCRIPTION
**Pull request by Br3nda**
Added jobseeker scenarios for people aged 18 and 19, with a dependent child.
closes  #56

### CODE REVIEW

File: openfisca_aotearoa/parameters/entitlements/social_security/jobseeker_support/minumum_years_in_nz.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-description: Minimum number of years in NZ before being eligible for Job Seeker
-reference: TODO MSD Policy https://www.workandincome.govt.nz/products/a-z-benefits/accommodation-supplement.html
+description: Minimum number of years in NZ before being eligible for Jobseeker Support
+reference: TODO
```
Comment (alexey0511) [inline openfisca_aotearoa/parameters/entitlements/social_security/jobseeker_support/minumum_years_in_nz.yaml]: should we have ref here?

File: openfisca_aotearoa/variables/demographics/relationships.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ class has_a_partner(Variable):
     value_type = bool
     entity = Person
     label = u"Is this person in a relationship?"
-    definition_period = MONTH  # This variable cannot change over time.
+    definition_period = MONTH  # This variable changes over time.
```
Comment (alexey0511) [inline openfisca_aotearoa/variables/demographics/relationships.py]: I don't understand why definition period changes ...

Comment (Br3nda) [reply openfisca_aotearoa/variables/demographics/relationships.py]: the definition period doesn't change.

has_a_partner is a Variable. Whether you have a partner or not will change over time.

File: openfisca_aotearoa/variables/demographics/relationships.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ class has_a_partner(Variable):
...
     reference = u"TODO"
```
Comment (alexey0511) [inline openfisca_aotearoa/variables/demographics/relationships.py]: another TODO?

Comment (Br3nda) [reply openfisca_aotearoa/variables/demographics/relationships.py]: yes - but it's an existing "TODO" in the codebase, not part of this PR

**Pull request merged**